stories/card.stories.js

```diff
@@ -82,10 +82,8 @@ export const Default = () => (
   </Grid>
 );
 
-Default.story = {
-  name: 'default',
-  parameters: { notes: { markdown: notes } }
-};
+Default.storyName = 'default';
+Default.parameters = { notes: { markdown: notes } };
```

Comment:
`.story = ` is also deprecated in favor of these top-level properties like `storyName` and `parameters`.